packages/client/tests/functional/relationMode/_matrix.ts

```diff
@@ -0,0 +1,64 @@
+import { defineMatrix } from '../_utils/defineMatrix'
+import { Providers } from '../_utils/providers'
+import { computeMatrix } from '../_utils/relationMode/computeMatrix'
+
+const RelationModeEnv = process.env.RELATION_MODE
+if (RelationModeEnv && RelationModeEnv !== 'prisma' && RelationModeEnv !== 'foreignKeys') {
+  throw new Error(`RELATION_MODE must be either "prisma" or "foreignKeys" but was "${RelationModeEnv}"`)
+}
+
+type RelationMode = 'prisma' | 'foreignKeys' | ''
+const relationMode: RelationMode = (RelationModeEnv as RelationMode) || ''
+
+// Note: testing 'SetDefault' requires a relation with a scalar field having the "@default" attribute.
+// If no defaults are provided for any of the scalar fields, a runtime error will be thrown.
```

Comment:
Are we testing that both database and emulated to confirm this?
(Should this maybe be a validation error instead?)

Reply:
SetDefault is invalid for `relationMode="prisma"`